internal/ethapi/transaction_args.go

```diff
@@ -53,6 +55,10 @@ type TransactionArgs struct {
 	// Introduced by AccessListTxType transaction.
 	AccessList *types.AccessList `json:"accessList,omitempty"`
 	ChainID    *hexutil.Big      `json:"chainId,omitempty"`
+
+	// Introduced by EIP-4844.
+	MaxFeePerBlobGas    *hexutil.Big  `json:"maxFeePerBlobGas"`
+	BlobVersionedHashes []common.Hash `json:"blobVersionedHashes,omitempty"`
```

Comment:
The json names of course need to stay the same, but IMO the internal field names should be the same as core.types.

Reply:
Done (this and the other comments).